src/lib/routes/admin-api/metrics.ts

```diff
@@ -28,19 +31,68 @@ class MetricsController extends Controller {
         this.get('/feature-toggles', this.deprecated);
         this.get('/feature-toggles/:name', this.deprecated);
 
-        // in use
-        this.post(
-            '/applications/:appName',
-            this.createApplication,
-            UPDATE_APPLICATION,
-        );
-        this.delete(
-            '/applications/:appName',
-            this.deleteApplication,
-            UPDATE_APPLICATION,
-        );
-        this.get('/applications/', this.getApplications);
-        this.get('/applications/:appName', this.getApplication);
+        this.route({
+            method: 'post',
+            path: '/applications/:appName',
+            handler: this.createApplication,
+            permission: UPDATE_APPLICATION,
+            middleware: [
+                openApiService.validPath({
+                    tags: ['admin'],
+                    operationId: 'metricsCreateApplication',
```

Comment:
Do we need the `metrics` prefix in the operationIds? Maybe we should add more tags instead.

Reply:
Hmm, good question, I guess createApplication is only done in metrics so I can change the opId

Reply:
Done